tool/tctl/common/tctl.go

```diff
@@ -754,39 +758,69 @@ func (a *AuthCommand) GenerateAndSignKeys(client *auth.TunClient) error {
 		return trace.Wrap(err)
 	}
 
-	certPath := a.genUser + "-cert.pub"
-	keyPath := a.genUser
-	pubPath := a.genUser + ".pub"
-
-	// --out flag
-	if a.outDir != "" {
-		if !utils.IsDir(a.outDir) {
-			if err = os.MkdirAll(a.outDir, 0770); err != nil {
+	switch a.outputFormat {
+	//
+	// dump user identity into a single file:
+	//
+	case IdentityFormatFile:
```

Comment:
`tctl` now dumps the identity to `stdout` by default but you can set `-o` to something else